hasher-matcher-actioner/webapp/src/pages/settings/ThreatExchangeSettingsTab.jsx

```diff
@@ -152,7 +152,7 @@ export default function ThreatExchangeSettingsTab() {
           <Spinner hidden={!loading} animation="border" role="status">
             <span className="sr-only">Loading...</span>
           </Spinner>
-          {datasets.length === 0
+          {!datasets || datasets.length === 0
```

Comment:
(also commented this on #665) `datasets` is initialized to `[]` so I didn't think that would be an issue. Does a bad API request make it undefined? It feels like this check might just hide an API level issues.